tests/utils_tests/test_autoreload.py

```diff
@@ -252,3 +252,32 @@ def test_resets_trans_real(self):
         self.assertEqual(trans_real._translations, {})
         self.assertIsNone(trans_real._default)
         self.assertIsInstance(trans_real._active, _thread._local)
+
+
+class RestartWithReloaderTests(SimpleTestCase):
+
+    def test_restart_with_reloader_with_manage_py(self):
+        executable = '/usr/bin/python'
+        argv = ['./manage.py', 'runserver']
+        with mock.patch('django.utils.autoreload.subprocess.call', return_value=0) as mock_call, \
+                mock.patch('django.utils.autoreload.sys.argv', argv), \
+                mock.patch('django.utils.autoreload.sys.executable', executable), \
+                mock.patch('django.utils.autoreload.sys.warnoptions', ['all']):
+
+            autoreload.restart_with_reloader()
+            mock_call.assert_called_once()
```

Comment:
I don't think we can use `assert_called_once()` yet since that's new in Python 3.6. With the change in `autoreload.py` reverted, both tests fail on Python 3.5 with `AttributeError: assert_called_once` while the first test will pass on Python 3.6.